tools/ci_build/github/linux/docker/scripts/training/azure_scale_set_vm_mount_gpt2_test_data.sh

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env bash
+if [ -d "/gpt2_data" ]; then
+sudo umount /gpt2_data
+fi
+if [ -d "/gpt2_data" ]; then
```

Comment:
nit: can these two blocks combined into one?

Reply:
the way an Azure shared disk is connected: 
- create a folder at the local machine
- mount the folder to the azure disk

thus these 2 block is to undo the above 2 steps. However, after the first block, if the folder does not exist, the second block is not to be executed.